src/overview/design-goals.md

```diff
@@ -43,3 +44,26 @@ We want to follow some conservative design heuristics to minimize risk and mista
 - Rely as much as possible on design components that are already proven in production environments.
 - Rely as much as possible on design components with adequate theoretical underpinnings and security analyses.
 - Minimize changes or variations on the above: strive to only alter existing work when necessary for overall design goals. For example, Zcash's privacy or issuance constraints are likely less common among existing PoS designs.
+
+
+# Non-goals
+
+These are not goals of the TFL design, either to simplify the scope of the initial design (aka [Out of Scope Goals](#out-of-scope-goals)), or because we believe some potential goal _should not_ be supported (aka [Anti-goals](#anti-goals)).
```

Comment:
```suggestion
These are not goals of the TFL design, either to simplify the scope of the initial design (aka [Out-of-Scope Goals](#out-of-scope-goals)), or because we believe some potential goal _should not_ be supported (a.k.a. [Anti-goals](#anti-goals)).
```